### PR TITLE
Feat(eos_cli_config_gen): Add IPv4 and IPv6 SR-TE address families

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Add SR-TE BGP address families - ipv4 one is required for WAN config and it was cheap to add v6 at the same time.

## Component(s) name

`arista.avd.eos_cli_config_gem`

## Proposed changes

```
      address_family_ipv4_sr_te:
        neighbors:
          - ip_address: <str>
            activate: <bool>
            route_map_in: <str>
            route_map_out: <str>
        peer_groups:
          - name: <str>
            activate: <bool>
            route_map_in: <str>
            route_map_out: <str>
[...]
      address_family_ipv6_sr_te:
        neighbors:
          - ip_address: <str>
            activate: <bool>
            route_map_in: <str>
            route_map_out: <str>
        peer_groups:
          - name: <str>
            activate: <bool>
            route_map_in: <str>
            route_map_out: <str>
```

## How to test

molecule

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml [hunk 1]
```diff
@@ -1035,6 +1035,46 @@ keys:
...
+                  type: str
+                  description: Inbound route-map name
+                route_map_out:
+                  type: str
+                  description: Outbound route-map name
+          peer_groups:
+            type: list
+            primary_key: name
+            convert_types:
+              - dict
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]: I don't think we need this for new models.

**Review by ClausHolbechArista — APPROVED** [review]
Looking good. Only a minor adjustment to the schema.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml (hunk 1 above)
Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]: Addressed in [c943be3](https://github.com/aristanetworks/ansible-avd/pull/3217/commits/c943be3d1a276b3e4152453d643efc0c380a9302)

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml [hunk 2]
```diff
@@ -1035,6 +1035,44 @@ keys:
...
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]:
```suggestion
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml [hunk 3]
```diff
@@ -1185,6 +1223,44 @@ keys:
...
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]:
```suggestion
```

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]: 52aeee1

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml (hunk 2 above)
Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml]: 52aeee1

**Review by emilarista — APPROVED** [review]
Tested, LGTM!

**Pull request merged**